gateway-service-impl/src/test/java/org/hypertrace/gateway/service/common/validators/function/AvgRateValidatorTest.java

```diff
@@ -316,7 +335,9 @@ public void avgRateFunctionWithEmptyAlias_shouldThrowIllegalArgException() {
                     .setLiteral(
                         LiteralConstant.newBuilder()
                             .setValue(
-                                Value.newBuilder().setLong(20L).setValueType(ValueType.LONG))))
+                                Value.newBuilder()
```

Comment:
Make sure to keep tests support long version - for the most part, this change should only add tests because it needs to maintain existing support (may be some exceptions for this for internal things that have changed).